src/Symfony/Component/Polyfill/bootstrap.php

```diff
@@ -0,0 +1,162 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+use Symfony\Component\Polyfill as p;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ */
+
+if (PHP_VERSION_ID < 50400) {
+    if (!function_exists('trait_exists')) {
+        function trait_exists($class, $autoload = true) {return p\Php54::trait_exists($class, $autoload);}
+    }
+    if (!function_exists('class_uses')) {
+        function class_uses($class, $autoload = true) {return p\Php54::class_uses($class, $autoload);}
+    }
+    if (!function_exists('hex2bin')) {
+        function hex2bin($data) {return p\Php54::hex2bin($data);}
+    }
+
+    // Take entropy from the OS for PHP session ids
+    if (!ini_get('session.entropy_file') && !ini_get('session.entropy_length')) {
+        ini_set('session.entropy_length', 32);
+
+        if (false !== @file_get_contents('/dev/urandom', false, null, -1, 1) ) {
```

Comment:
in fact I'm going to remove these ini_setting: it's reducing the entropy for auto conf, not a good